net/ipfixprobe/Makefile

```diff
@@ -39,18 +21,22 @@ include $(INCLUDE_DIR)/package.mk
 define Package/ipfixprobe
   SECTION:=net
   CATEGORY:=Network
-  #SUBMENU:=P4_exporter
-  DEPENDS:=+libpcap +libstdcpp
+  SUBMENU:=NEMEA
```

Comment:
I agree with all the changes except this line - we need to discuss this.
Generally, ipfixprobe is totally independent from NEMEA to minimize its size.
Therefore, it's a question if we want to put it there...
From my opinion, it belongs to Network like, e.g., softflowd...

Reply:
Well, that could be changed anytime. So, if it is independed, then the submenu for ipfixprobe is dropped.